.github/bot/src/index.ts

```diff
@@ -1,7 +1,7 @@
 import { setFailed } from "@actions/core";
 import { EVENTS, ERRORS } from "./utils";
 import { context, getOctokit } from "@actions/github";
-import { GITHUB_TOKEN } from "src/utils/Constants";
+import { GITHUB_TOKEN } from "./utils/Constants";
```

Comment:
I have had issues with relative paths in the past, is there any particular reason you changed from absolute to relative?